Examples/Scripts/Python/seeding.py

```diff
@@ -280,6 +280,7 @@ def customLogLevel(custom: acts.logging.Level = acts.logging.INFO):
             outputProtoTracks="prototracks_estimated",
             trackingGeometry=trackingGeometry,
             magneticField=field,
+            initialVarInflation=initialVarInflation,            
             **acts.examples.defaultKWArgs(
```

Comment:
The default `initialVarInflation` is `None`, which probably isn't understood by `TrackParamsEstimationAlgorithm()`. You can easily fix it by moving the setting inside the `defaultKWArgs()`.
```suggestion
            **acts.examples.defaultKWArgs(
               initialVarInflation=initialVarInflation,            
```

Reply:
Ok I moved it.

Reply:
Ok I moved it.